blockstack_client/client.py

```diff
@@ -77,38 +54,35 @@ def session(conf=None, config_path=CONFIG_PATH, server_host=None, server_port=No
     Returns the API proxy object.
     """
 
-    if conf is None and config_path is not None:
+    if not conf and config_path:
```

Comment:
I'd prefer keeping this as `conf is None` instead of `not conf`.  The reason is that I'm trying to avoid situations where class operator overloading (e.g. of `__nonzero__()` or `__bool__()`) could introduce hard-to-decipher bugs and add accidental complexity.  If `conf` is neither `None` nor a `dict` (or something that behaves like a `dict`), I'd prefer for an exception to be raised.